README.md

```diff
@@ -0,0 +1,197 @@
+
+The transformer will also generate a file named `utils.js` inside the wrapper class directory, which contains utilities used by all the wrapper class files.
+
+Note that webidl2js works best when there is a single transformer instance that knows about as many files as possible. This allows it to resolve type references, e.g. when one operation has an argument type referring to some other interface.
+
+## Features
+
+webidl2js is implementing an ever-growing subset of the Web IDL specification. So far we have implemented:
+
+- Interface types, including:
+  - Inheritance
+  - Attributes
+  - Operations
+  - Constants
+  - Stringifiers
+  - `iterable<>` declarations
+  - Class strings (with the semantics of [heycam/webidl#357](https://github.com/heycam/webidl/pull/357))
+- Dictionary types
+- Union types
+- Callback function types, somewhat
+- Nullable types
+- `sequence<>` types
+- `record<>` types
+- `Promise<>` types
+- `FrozenArray<>` types
+- `typedef`s
+- Partial interfaces and dictionaries
+- Basic types (via [webidl-conversions](https://github.com/jsdom/webidl-conversions))
+- Mixins, i.e. `implements` (with a [notable bug](https://github.com/jsdom/webidl2js/issues/49))
+- Overload resolution (although [tricky cases are not easy on the implementation class](https://github.com/jsdom/webidl2js/issues/29))
+- Variadic arguments
+- `[Clamp]`
+- `[Constructor]`
+- `[EnforceRange]`
+- `[Exposed]` and `[NoInterfaceObject]` (by exporting metadata on where/whether it is exposed)
+- `[PutForwards]`
+- `[Replaceable]`
+- `[SameObject]` (automatic caching)
+- `[TreatNullAs]`
+- `[Unforgeable]`
+- `[Unscopeable]`
+
+Notable missing features include:
+
+- Legacy platform objects, i.e. those using `getter`/`setter`/`deleter` declarations ([in progress](https://github.com/jsdom/webidl2js/pull/48))
+- Namespaces
+- Enumeration types ([#28](https://github.com/jsdom/webidl2js/issues/28))
+- Callback interfaces
+- Named constructors
+- `maplike<>` and `setlike<>`
+- `[AllowShared]`
+- `[Default]` (for `toJSON()` operations)
+- `[Global]` and `[PrimaryGlobal]`'s various consequences, including the named properties object and `[[SetPrototypeOf]]`
+- `[LegacyArrayClass]`
+- `[LegacyUnemerableNamedProperties]`
+- `[LegacyWindowAlias]`
+- `[LenientSetter]`
+- `[LenientThis]`
+- `[NamedConstructor]`
+- `[OverrideBuiltins]`
+- `[SecureContext]`
+- `[TreatNonObjectAsNull]`
+
+## Nonstandard Extended Attributes
+
+A couple of non-standard extended attributes are baked in to webidl2js.
+
+### `[Reflect]`
+
+The `[Reflect]` extended attribute is implemented to call `this.getAttribute` or `this.setAttribute` and process the input our output using [webidl-html-reflector](https://github.com/domenic/webidl-html-reflector), generating both a getter and a setter. If `[Reflect]` is specified, the implementation class does not need to implement any getter or setter logic.
```

Comment:
Ah, I forgot that this switched when @Sebmaster created webidl2js instead of my old https://github.com/domenic/webidl-class-generator. (Which I should probably tombstone...)